recipes-qt/qt5/qtsystems_git.bb

```diff
@@ -1,3 +1,5 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```

Comment:
Should be :prepend and BPN, but more importantly it isn't needed at all, will drop this when cherry-picking to master-next.